Basic/ClientDriven/Assets/Prefabs/Ingredient.prefab

```diff
@@ -641,9 +655,9 @@ MonoBehaviour:
   SyncRotAngleX: 1
```

Comment:
I can't remember, was looking up and down replicated as well? Or just character position and animations?

Reply:
Synced offline. Will look into adding looking rotation to a follow-up update.